schemas/basicDataTypes/array/additionalItems.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "array",
+    "items": [
+      {
+        "type": "number"
+      },
+      {
+        "type": "string"
+      }
+    ],
+    "additionalItems": { "type": "boolean" }

```

Comment:
Since `additionalItems` keyword is meaningless with list validation type of lists
https://json-schema.org/understanding-json-schema/reference/array.html#items,
used the keyword with tuple validation example instead!